sys/include/net/gcoap.h

```diff
@@ -400,8 +420,31 @@ extern "C" {
 /**
  * @brief   Server port; use RFC 7252 default if not defined
  */
-#ifndef CONFIG_GCOAP_PORT
 #define CONFIG_GCOAP_PORT              (5683)
```

Comment:
leave the `#ifndef` in place, add a new one for `CONFIG_GCOAPS_PORT`

Reply:
Done.